frontend/Dockerfile

```diff
@@ -16,7 +16,7 @@ ARG API_URL_BROWSER=/
 ENV API_URL=${API_URL}
 ENV API_URL_BROWSER=${API_URL_BROWSER}
 
-RUN yarn build
+RUN npm run build
```

Comment:
Reading the README makes me chuckle, because I have NEVER gotten it to work outside docker-compose successfully with nuxt development server. I'll change it to match npm usage though.